src/generators/plugin.ts

```diff
@@ -153,71 +157,13 @@ export default class Plugin extends Generator {
 
     const final = Object.assign({}, pjson, updated);
 
-    if (!this.answers.internal) {
-      // If we are building a 3PP plugin, we don't want to set defaults for these properties.
-      // We could ask these questions in the prompt, but that would be too many questions for a good UX.
-      // We want developers to be able to quickly get up and running with their plugin.
-      delete final.homepage;
-      delete final.repository;
-      delete final.bugs;
-
-      // 3PP plugins don't need these tests.
-      delete final.scripts['test:json-schema'];
-      delete final.scripts['test:deprecation-policy'];
-      delete final.scripts['test:command-reference'];
-      final.scripts.posttest = 'yarn lint';
-
-      // 3PP plugins don't need these either.
-      // Can't use the class's this.fs since it doesn't delete the directory, just the files in it.
-      fs.rmSync(this.destinationPath('./schemas'), { recursive: true });
-      fs.rmSync(this.destinationPath('./.git2gus'), { recursive: true });
-      fs.rmSync(this.destinationPath('./.github'), { recursive: true });
-      fs.rmSync(this.destinationPath('./command-snapshot.json'));
-      fs.rmSync(this.destinationPath('./CODE_OF_CONDUCT.md'));
-      fs.rmSync(this.destinationPath('./SECURITY.md'));
-
-      // Remove /schemas from the published files.
-      final.files = final.files.filter((f) => f !== '/schemas');
-
-      this.fs.delete(this.destinationPath('./.circleci/config.yml'));
-      this.fs.copy(
-        this.destinationPath('./.circleci/external.config.yml'),
-        this.destinationPath('./.circleci/config.yml')
-      );
-
-      const nycConfig = readJson<NYC>(path.join(this.env.cwd, '.nycrc'));
-      const codeCoverage = Number.parseInt(this.answers.codeCoverage.replace('%', ''), 10);
```

Comment:
Since we are removing this we can remove the codecov question code and messages:
https://github.com/salesforcecli/plugin-dev/blob/05c60cd17064799066e22f2229b360537e714d65/src/generators/plugin.ts#L96-L102